dotnet/src/Experimental/Agents/Extensions/AssistantsKernelExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Experimental.Agents.Exceptions;
+
+namespace Microsoft.SemanticKernel.Experimental.Agents.Extensions;
+
+internal static class AssistantsKernelExtensions
```

Comment:
Cosmetic - `AgentKernelExtensions` or maybe just `KernelExtensions`?

Reply:
This seems to be related to specific OpenAI Assistant stuff, and is internal only. Leaving it for now but we can change it if needed in a follow up.